OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix root files

# Description
lint fix for the files below:
![CleanShot 2023-10-26 at 00 39 09@2x](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/41e72af4-788a-46f3-bab5-edf4af3f451b)


## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin user
4. **Record a video to ensure all functions are operating normally.**




No need to test the eslint, I have tested, below is the screenshot.

![CleanShot 2023-10-26 at 00 41 58@2x](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/aa27b135-eef0-4f40-9415-d20345289167)
